CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+Esri welcomes contributions from anyone and everyone. Please see our [guidelines for contributing](https://github.com/esri/contributing).
+
+### Before filing an issue
+
+Please take a look at [previous issues](https://github.com/Esri/arcgis-python-api/issues?q=is%3Aissue+is%3Aclosed) that resolve common problems.
+
+If you're just looking for help, you'll probably attract the most eyes if you post in the [GeoNet Forums](https://community.esri.com/groups/arcgis-python-api/) or the [GIS Stackexchange](https://gis.stackexchange.com/questions/tagged/arcgis-python-api).
```

Comment:
Suggest removing stack exchange. Esri cannot moderate that forum and we cannot endorse content presented there.